.github/workflows/test.yml

```diff
@@ -32,5 +32,5 @@ jobs:
           curl -X POST \
             -H "Authorization: Bearer ${{ secrets.REGRESSION_TEST }}" \
             -H "Accept: application/vnd.github.v3+json" \
-            "https://api.github.com/repos/MaximIntegratedAI/ai8x-regression/dispatches" \
+            "https://api.github.com/repos/analogdevicesinc/ai8x-regression/dispatches" \
```

Comment:
Ai8x-regression repository has not been moved to analogdevicesinc organization yet.

Reply:
Thanks for catching this